src/js/view/utils/views-manager.js

```diff
@@ -56,7 +56,7 @@ function scheduleResponsiveRedraw() {
 
 function onOrientationChange() {
     views.forEach(view => {
-        if (view.model.get('visibility') >= 0.75) {
+        if (!view.model.get('audioMode') && view.model.get('controls') && view.model.get('visibility') >= 0.75) {
```

Comment:
Probably worth assigning `view.model` to a variable since it's used 4 times. It might also look neater to return early and reduce nesting by 1 indentation